include/emp/base/array.hpp

```diff
@@ -25,6 +25,14 @@
 #include "assert.hpp"
 #include "../meta/TypeID.hpp"
 
+namespace emp {
+
+  /// Determine the size of a built-in array.
```

Comment:
can we move this to meta/meta? ( https://github.com/devosoft/Empirical/blob/master/include/emp/meta/meta.hpp )